app/src/main/java/se/rebeccazadig/bokholken/login/UserViewModel.kt

```diff
@@ -28,11 +27,11 @@ class UserViewModel : ViewModel() {
         inProgress.value = true
 
```

Comment:
Check if all fields are empty and handle the case when all fields are empty, like showing a toast

Reply:
Im not sure what you mean

Reply:
Erkan wrote that it should be impossible to create a user with empty name, city, contact. The app user should also be informed why the app doesn't allow to create such a user.